### ISSUE TITLE
data augmentation and cross validation

### ISSUE THREAD

**Issue by HIN0209** [matthew-sochor-zz/transfer#12]
Hi, I really like this project. Being a data-science novice, I have a question.

I have two-classed images (170 and 304 each, the sum being 474) and augmented. Training was successfully finished. The sum of the numbers in confusion matrix was 1,290, which appears to me that the augmented images were used as test data. I think that augmented data should be used only for training, but only original data must be used for testing. Am I missing something, please educate? 

My config.yaml is as follows:
- api_port: 5000
  architecture: inception_v3
  augmentations:
    channel_shift_range: 0.1
    cval: 0.0
    featurewise_center: false
    featurewise_std_normalization: false
    fill_mode: nearest
    height_shift_range: 0.0
    horizontal_flip: false
    rescale: null
    rotation_range: 0
    rounds: 10
    samplewise_center: false
    samplewise_std_normalization: false
    shear_range: 0.0
    vertical_flip: false
    width_shift_range: 0.2
    zca_whitening: false
    zoom_range: [1.0, 1.0]
  batch_size: 16
  best_weights: [./xxx/weights/xxx-inception_v3-weights--kfold-0-round-1-kfold-0-improvement-59-0.9797.hdf5]
  categories: [xxx, yyy]
  category_rounds: {xxx: 1, yyy: 2}
  conv_dim: 8
  cycle: 3
  final_cutoff: 80
  img_dim: 299
  img_path: ./xxx
  img_size: 1
  is_array: true
  is_augmented: true
  is_final: false
  is_pre_model: true
  is_split: false
  kfold: 5
  kfold_every: false
  last_weights: [./xxx/weights/xxx-inception_v3-weights--kfold-0-round-1.hdf5]
  learning_rate: 0.001
  learning_rate_decay: 1.0e-06
  model_round: 1
  name: xxx
  number_categories: 2
  path: ./xxx
  plot: true
  rounds: 30
  seed: 3620
  server_weights: null
